docs/api/browser-window-proxy.md

```diff
@@ -50,4 +50,4 @@ The `BrowserWindowProxy` object has the following instance properties:
 
 #### `win.closed`
 
-A Boolean that is set to true after the child window gets closed.
+A `Boolean` that is set to true after the child window gets closed.
```

Comment:
> It's the same syntax like in method returns

The docs linter still doesn't support it though 😆 Two different regexps.  Just querying whether it is a good change, if so we can modify the docs linter.